build-tools/stylelint/license-headers.js

```diff
@@ -0,0 +1,56 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This is copied from https://github.com/cloudscape-design/components/blob/main/build-tools/stylelint/license-headers.js